walk-in-interview/src/main/webapp/new-job/index.html

```diff
@@ -12,48 +12,48 @@
 <body>
 
     <div class="header-container">
-        <input id="new-job-cancel">
-        <div id="new-job-page-title"></div>
-        <input id="new-job-submit">
+        <input id="cancel">
+        <div id="page-title"></div>
+        <input id="submit">
     </div>
 
     <div class="content-container">
         <div class="content-container">
-            <input class="job-field" id="new-job-title" />
-            <textarea class="job-field" id="new-job-description"></textarea>
+            <input class="job-field" id="title" />
+            <textarea class="job-field" id="description"></textarea>
 
             <div class="job-field">
-                <input id="new-job-address" />
-                <input id="new-job-postal-code" />
+                <input id="address" />
+                <input id="postal-code" />
             </div>
 
             <div class="job-field">
-                <div id="new-job-requirements-title"></div>
-                <ul id="new-job-requirements-list"></ul>
+                <div id="requirements-title"></div>
+                <ul id="requirements-list"></ul>
             </div>
 
             <div class="job-field">
-                <div id="new-job-pay-title"></div>
-                <select id="new-job-pay-frequency"></select>
-                <div id="new-job-pay-range">
-                    <input id="new-job-pay-min">
+                <div id="pay-title"></div>
+                <select id="pay-frequency"></select>
+                <div id="pay-range">
+                    <input id="pay-min">
                     -
-                    <input id="new-job-pay-max">
+                    <input id="pay-max">
                 </div>
             </div>
 
             <div class="job-field">
-                <div id="new-job-duration-title"></div>
-                <select id="new-job-duration"></select>
+                <div id="duration-title"></div>
+                <select id="duration"></select>
             </div>
 
             <div class="job-field">
-                <div id="new-job-expiry-title"></div>
-                <input id="new-job-expiry">
+                <div id="expiry-title"></div>
+                <input id="expiry">
             </div>
         </div>
 
-        <div class="error-message" id="new-job-error-message"></div>
+        <div class="error-message" id="error-message"></div>
 
         <div hidden>
             <li id="requirement-element-template">
```

Comment:
maybe add a comment to indicate that this hidden div can be used as a template?

Reply:
okay sure